tests/input_data/configuration/test_build_mortality.py

```diff
@@ -9,7 +9,9 @@
 @pytest.mark.parametrize("sexes", [1, 2])
 def test_add_mortality_data(ihme, sexes):
     ec = make_execution_context(model_version_id=265976, gbd_round_id=5)
+    ec.policies = dict(use_weighted_age_group_midpoints=0)
```

Comment:
Ugg. This is awful. I never meant for this  to be an attribute on ExecutionContext. I'll make a PR to clean this up.